projects/steering/inc/steering_events.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+typedef enum {
+  // Digital
+  STEERING_INPUT_HORN_EVENT = 0,
+  STEERING_RADIO_PPT_EVENT,
+  STEERING_HIGH_BEAM_FORWARD_EVENT,
+  STEERING_HIGH_BEAM_REAR_EVENT,
+  STEERING_REGEN_BRAKE_EVENT,
+  STEERING_INPUT_CC_TOGGLE_PRESSED_EVENT,
+  NUM_DIGITAL_STEERING_EVENTS,
+} SteeringDigitalEvent;
+
+typedef enum {
+  // Analog
+  STEERING_CONTROL_STALK_EVENT_LEFT_SIGNAL = 7,

```

Comment:
Should probably be explicit i.e.
```suggestion
  STEERING_CONTROL_STALK_EVENT_LEFT_SIGNAL = NUM_DIGITAL_STEERING_EVENTS + 1,

```